onnxruntime/core/framework/parallel_executor.cc

```diff
@@ -258,8 +258,11 @@ void ParallelExecutor::EnqueueNode(size_t p_node_index, const SessionState& sess
   executor_pool_->Schedule([this, p_node_index, &session_state, &logger]() {
     try {
       ParallelExecutor::RunNodeAsync(p_node_index, std::cref(session_state), std::cref(logger));
-    } catch (...) {
+    } catch (::onnxruntime::OnnxRuntimeException & ex) {
```

Comment:
Don't we also want to stop execution? Logging seems insufficient given that.

I would also expect the log level to be ERROR.

Reply:
Stopping, yes, as that's what the sequential one does as well. But this would likely involve propagating the exception within the thread to the main thread and I wasn't sure how to do this here. That's why I've declared this as an intermediate step.

Log level ERROR makes sense.

Reply:
#1262 will hopefully address this more fully.